spec.html

```diff
@@ -19896,8 +19853,7 @@ <h1>Runtime Semantics: InstantiateArrowFunctionExpression</h1>
         1. If _name_ is not present, set _name_ to *""*.
         1. Let _scope_ be the LexicalEnvironment of the running execution context.
         1. Let _sourceText_ be the source text matched by |ArrowFunction|.
-        1. Let _parameters_ be CoveredFormalsList of |ArrowParameters|.
-        1. [id="step-arrowfunction-evaluation-functioncreate"] Let _closure_ be OrdinaryFunctionCreate(%Function.prototype%, _sourceText_, _parameters_, |ConciseBody|, ~lexical-this~, _scope_).
+        1. [id="step-arrowfunction-evaluation-functioncreate"] Let _closure_ be OrdinaryFunctionCreate(%Function.prototype%, _sourceText_, |ArrowParameters|, |ConciseBody|, ~lexical-this~, _scope_).
```

Comment:
This is the only change which is suspicious, but as far as I can tell it all works: although this has the effect of changing the `[[FormalParameters]]` slot of arrow functions from containing an `|ArrowFormalParameters|` to containing a `|CoverParenthesizedExpressionAndArrowParameterList|`, it appears to be the case that all consumers of `[[FormalParameters]]` are already equipped to handle `|CoverParenthesizedExpressionAndArrowParameterList|` correctly.